File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P04Ex_FOR_Loop/P08_TennisRanklist.java
package P04Ex_FOR_Loop;

import java.util.Scanner;

public class P08_TennisRanklist {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int tournaments = Integer.parseInt(scanner.nextLine());
        int startPoints = Integer.parseInt(scanner.nextLine());
        String level = "";
        int points = 0, wins = 0;
        for (int count = 0; count < tournaments; count++) {
            level = scanner.nextLine();
            switch (level) {
                case "W":
                    points += 2000;
                    wins++;
                    break;
                case "F":
                    points += 1200;
                    break;
                case "SF":
                    points += 720;
                    break;
            }
        }
        System.out.printf("Final points: %d%n", points + startPoints);
        System.out.printf("Average points: %.0f%n", Math.floor(points * 1.0 / tournaments));
        System.out.printf("%.2f%%", wins * 100.0 / tournaments);
    }
}

File: JavaScript_Basic_04_2023_SolutionsOfProblems/04.Ex_PB-JS-For-Loop/P08_TennisRanklist.js
function tennisRanklist(input) {
    let tournaments = Number(input[0]);
    let startPoints = Number(input[1]);
    let points = 0, winTournaments = 0;
    for (i = 1; i <= tournaments; i++) {
        let result = input[i + 1];
        switch (result) {
            case "W":
                points += 2000;
                winTournaments++;
                break;
            case "F":
                points += 1200;
                break;
            case "SF":
                points += 720;
                break;
        }
    }
    console.log(`Final points: ${startPoints + points}`);
    console.log(`Average points: ${Math.floor(points / tournaments)}`);
    console.log(`${(winTournaments * 100 / tournaments).toFixed(2)}%`);
}
tennisRanklist(["4",
    "750",
    "SF",
    "W",
    "SF",
    "W"])
